### PR TITLE
chưa làm trường hợp */ trước, +- sau

### PR DESCRIPTION
**Pull request by 1112134**

### CODE REVIEW

File: test.go [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+	scanner := bufio.NewScanner(os.Stdin)
+	fmt.Print("> ")
+	for scanner.Scan() {
+		text := scanner.Text()
+		err := eval(text)
+		if err != nil {
+			fmt.Println(err)
+			continue
+		}
+		fmt.Print("> ")
```
Comment (havinhdat) [inline test.go:22]: we should print `>` even if there was error

File: test.go [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+	}
+}
+
+func parse(text string) (aFloat []float64, aOp []string, err error) {
+	expr := strings.Split(text, " ")
```
Comment (havinhdat) [inline test.go:27]: `strings.Fields` would cover more edge cases
https://golang.org/pkg/strings/#Fields

File: test.go [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+	if err != nil {
+		return
+	}
+
+	total = aFloat[0]
+
+	for i := 0; i < len(aOp); i++ {
+		switch aOp[i] {
+		case "+":
+			total = total + aFloat[i+1]
```
Comment (havinhdat) [inline test.go:85]: access slice without checking it's length could cause index out of range runtime error
